RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,38 +8,69 @@
 
 import Cocoa
 
+public protocol EntitiesViewDataSource : NSObjectProtocol {
+    func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    func entitiesView(tableView:NSTableView, viewForTableColmn tableColumn: NSTableColumn?, row: Int) -> NSView?
+}
+
+public protocol EntitiesViewDelegate : NSObjectProtocol {
+    func addEntityInEntitiesView(entitiesView:EntitiesView)
+    func removeEntityInEntitiesView(entitiesView:EntitiesView)
+    func entitiesViewSelectionDidChange(notification: NSNotification)
+}
+
 @IBDesignable
-class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource {
+public class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource {
     
     @IBOutlet var tableView:NSTableView!
-    
     @IBOutlet var addButton:NSButton!
     @IBOutlet var removeButton:NSButton!
     
-    override func nibDidLoad() {
+    var dataSource:EntitiesViewDataSource?
+    var delegate:EntitiesViewDelegate?
+    
+    override public func nibDidLoad() {
         super.nibDidLoad()
     }
     
+    override public func prepareForInterfaceBuilder() {
+        super.prepareForInterfaceBuilder()
+    }
+    
     //MARK: - NSTableViewDataSource
-    func numberOfRowsInTableView(tableView: NSTableView) -> Int {
+    public func numberOfRowsInTableView(tableView: NSTableView) -> Int {
```

Comment:
why did this need to change to public? an accessing class doesn't need to know that this method is implemented